examples/char/index.html

```diff
@@ -49,8 +49,7 @@
     <body>
         <button id="add-counter" type="button">add counter</button>
         <div id="container">
-            
+
         </div>
-        <script type="text/javascript" src="index.js"></script>
```

Comment:
Are we using the fancy webpack thing where it automatically includes `index.js`?